src/runtime/vendor/github.com/containerd/cgroups/systemd.go

```diff
@@ -105,13 +106,22 @@ func (s *SystemdController) Create(path string, _ *specs.LinuxResources) error {
 	once.Do(checkDelegate)
 	properties := []systemdDbus.Property{
 		systemdDbus.PropDescription("cgroup " + name),
-		systemdDbus.PropWants(slice),
 		newProperty("DefaultDependencies", false),
 		newProperty("MemoryAccounting", true),
 		newProperty("CPUAccounting", true),
 		newProperty("BlockIOAccounting", true),
 	}
 
+	if strings.HasSuffix(name, ".slice") {
+		properties = append(properties,
+			systemdDbus.PropWants(slice),
+		)
+	} else if strings.HasSuffix(name, ".scope") {
+		properties = append(properties,
+			systemdDbus.PropSlice(slice),
+			systemdDbus.PropPids(uint32(os.Getpid())),
+		)
+	}
```

Comment:
Putting this comment here but it's not directly to the above piece of code...

1. Have we sent a patch upstream to fix that?
2. I'd prefer if we'd create a temporary `containerd/cgroups` fork that carries those 2 patches and use that for as our vendored package (through the `replace` function in `runtime/go.mod`). Once/if those patches get accepted upstream we can switch back to the upstream version. If we agree with that approach we can quickly create a `cgroups` fork under the kata-containers organization.